lib/ansible/modules/cloud/ovirt/ovirt_templates.py

```diff
@@ -64,6 +64,25 @@
         description:
             - "Boolean indication whether to allow partial registration of a template when C(state) is registered."
         version_added: "2.4"
+    vnic_profile_mappings:
+        description:
+            - "Mapper which maps an external virtual NIC profile to one that exists in the engine when C(state) is registered."
```

Comment:
But currently the engine only supports specific attributes (which are mainly name).
I think that this might be confusing since if the user will add new properties those will not be mapped in the engine.